SPECS/initramfs/initramfs.spec

```diff
@@ -101,14 +102,20 @@ elif [ -d %{_localstatedir}/lib/rpm-state/initramfs/pending ]; then \
         fi \
     done; \
 fi \
+%grub2_post
 %removal_action
 
 %posttrans
 echo "initramfs" %{version}-%{release} "posttrans" >&2
 %removal_action
 mkinitrd -q
-# Move initrd generated for kernel-mshv to /boot/efi, where linuxloader expects to find it
-mv /boot/initrd.img-*mshv* /boot/efi/ >/dev/null 2>&1 || :
+# Copy initrd generated for kernel-mshv to /boot/efi, where linuxloader expects to find it
+cp /boot/initrd.img-*mshv* /boot/efi/
+if [ $? -ne 0 ]; then
+    echo "WARNING: ESP partition (/boot/efi) is likely full!"
```

Comment:
I think this should be a fatal error, if we allow rpm installation to continue the system could wind up in an unbootable state...

Reply:
But this is posttrans, right? Isn't the damage already done?

Reply:
Hold on, we just removed support for the old linuxloader https://dev.azure.com/mariner-org/mariner/_git/CBL-Mariner-Proprietary/pullrequest/17113. We now can rely upon the initrd present in /boot and remove this posttrans logic to duplicate it.

Reply:
Removing this block of logic entirely, no longer needed.